### PR TITLE
[Examples] Improve Contentful example starter and docs

### PR DESCRIPTION
**Pull request by stefanjudis**
This pull requests adds a Contentful setup script and improves the setup instructions by adding some screenshots. :) 

@lfades :)

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12s | 12.1s | ⚠️ +147ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.978 | 2.023 | ⚠️ +0.05 |
| / avg req/sec | 1263.9 | 1235.56 | -28.34 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.039 | 1.027 | -0.01 |
| /error-in-render avg req/sec | 2405.49 | 2433.21 | ⚠️ +27.72 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 926 B | 926 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 924 B | 924 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.8s | 12.8s | ⚠️ +3ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _error.js | 557 kB | 557 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 557 kB | 557 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 588 kB | 588 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.1s | -35ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.956 | 1.981 | ⚠️ +0.03 |
| / avg req/sec | 1278.21 | 1261.74 | -16.47 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.019 | 1.026 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2453.51 | 2437.57 | -15.94 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 926 B | 926 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 924 B | 924 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.1s | -808ms |
| nodeModulesSize | 55.2 MB | 55.2 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.28 kB | 6.28 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5232.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.82 kB | 4.82 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.88 kB | 6.88 kB | ✓ |
| framework.HA..dule.js gzip | 39.2 kB | 39.2 kB | ✓ |
| Overall change | 51.7 kB | 51.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.13 kB | 3.13 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | stefanjudis/next.js canary | Change |
| - | - | - | - |
| _error.js | 557 kB | 557 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 557 kB | 557 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 588 kB | 588 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

File: examples/cms-contentful/README.md [hunk 1]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
+This project includes a setup script which you can use set up the content model that are expected by the source code.
+
+In your Contentful dashboard go to **Settings > General Settings** and copy the **Space ID**.
+
+Next, go to **Settings > API > Content management tokens** and create a new token by clicking **Generate personal token**. This token has the same access rights as the logged in user. **Do not share it publicly**, you will only use it to set up your space and can delete it afterwards.
+
+With the space ID and management access token at hand run the following command:
+
+```
+CONTENTFUL_SPACE_ID=YOUR_SPACE_ID CONTENTFUL_MANAGEMENT_TOKEN=XXX npm run setup
```
Comment (lfades) [inline examples/cms-contentful/README.md:51]: Can we make sure to support Windows here too :pray:

Comment (lfades) [reply examples/cms-contentful/README.md:51]: Also, let's mention that the same is possible with `yarn`

File: examples/cms-contentful/README.md [hunk 2]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
+After creating an account, create a new empty **space** from the [dashboard](https://app.contentful.com/) and assign to it any name of your liking.
 
-### Step 2. Create an `author` content type
+### Step 2. Create a content model
+
+The [content model](https://www.contentful.com/developers/docs/concepts/data-model/) defines the data structures of your application/websites. The structures are flexible and you can tailor them to your needs.
+
+For this example you need to create a content model that defines an author and a post content type. **You can continue creating these two by running a script or doing it manually** to familiarize yourself with the Contentful user interface.
+
+#### Run a script to creat the content model
```
Comment (lfades) [inline examples/cms-contentful/README.md:40]:
```suggestion
#### Run a script to create the content model
```

File: examples/cms-contentful/README.md [hunk 3]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
+ ✔ Push content to destination space
+   ✔ Connecting to space (1s)
+   ...
+   ...
+   ...
+```
+
+#### Create the content model manually
+
+##### Create an `Author` content type
```
Comment (lfades) [inline examples/cms-contentful/README.md:88]: These titles are very nested and it's hard to see them as titles, maybe we can restructure things a little bit?

File: examples/cms-contentful/README.md [hunk 4]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
 - `date` - **Date and time** field
 - `slug` - **Text** field. You can optionally go to the settings of this field, and under **Appearance**, select **Slug** to display it as a slug of the `title` field.
-- `author` - **Reference** field
+- `author` - **Reference** field (type **one reference**)
 
 Save the content type and continue.
 
+### Step 3. Validate your content model
+
+After setting up the content model (either manually or by running `npm run setup`), it should look as follows.
```
Comment (lfades) [inline examples/cms-contentful/README.md:121]:
```suggestion
After setting up the content model (either manually or by running `npm run setup` or `yarn setup`), it should look as follows.
```

File: examples/cms-contentful/README.md [hunk 5]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
-### Step 2. Create an `author` content type
+### Step 2. Create a content model
+
+The [content model](https://www.contentful.com/developers/docs/concepts/data-model/) defines the data structures of your application/websites. The structures are flexible and you can tailor them to your needs.
+
+For this example you need to create a content model that defines an author and a post content type. **You can continue creating these two by running a script or doing it manually** to familiarize yourself with the Contentful user interface.
+
+#### Run a script to creat the content model
+
+This project includes a setup script which you can use set up the content model that are expected by the source code.
```
Comment (lfades) [inline examples/cms-contentful/README.md:42]:
```suggestion
This project includes a setup script which you can use to set up the content model expected by the source code.
```

File: examples/cms-contentful/README.md [hunk 6]
```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
...
 
-After creating an account, create a new **space** from the [dashboard](https://app.contentful.com/) and assign to it any name of your liking.
+After creating an account, create a new empty **space** from the [dashboard](https://app.contentful.com/) and assign to it any name of your liking.
 
-### Step 2. Create an `author` content type
+### Step 2. Create a content model
+
+The [content model](https://www.contentful.com/developers/docs/concepts/data-model/) defines the data structures of your application/websites. The structures are flexible and you can tailor them to your needs.
+
+For this example you need to create a content model that defines an author and a post content type. **You can continue creating these two by running a script or doing it manually** to familiarize yourself with the Contentful user interface.
```
Comment (lfades) [inline examples/cms-contentful/README.md:38]:
```suggestion
For this example you need to create a content model that defines an author and a post content type. **You can create these two by running a script or by doing it manually** to familiarize yourself with the Contentful user interface.
```

**Review by lfades — CHANGES_REQUESTED** [review]
:100: 

Left some suggestions :pray:

File: examples/cms-contentful/README.md (hunk 1 above)
Comment (Timer) [reply examples/cms-contentful/README.md:51]:
```suggestion
npx cross-env CONTENTFUL_SPACE_ID=YOUR_SPACE_ID CONTENTFUL_MANAGEMENT_TOKEN=XXX npm run setup
```

File: examples/cms-contentful/README.md (hunk 3 above)
Comment (Timer) [reply examples/cms-contentful/README.md:88]: The rendered view looks good to me IMO.

**Review by Timer — APPROVED** [review]

**Pull request merged**